### DISCUSSION TITLE
Question about Automating Labels using GitHub Action

### DISCUSSION THREAD

**Discussion by Rahma-AlWadhahi** [EddieHubCommunity/support#1211] [Q&A] [answered]
### Description

I had a talk recently about **[Automation with GitHub Actions](https://youtu.be/PXEPoOA3Mn0)** and I was glad to implement something similar to what Eddie explained in this video. [Sync GitHub Issue Labels with a json file and a GitHub Action](https://www.youtube.com/watch?v=HTFf0P2DlvQ)

I was wondering if we can have one repository which has the JSON file and apply it to all the repositories that a person or an organization owns.

For example, I have a repository1 that is recently created and has `labels.json` and `labels.yml`. In addition, I have other old repositories in my account. Is there a way I can use `repository1` and apply the labels without needing to copy the JSON file in all the repositories?

As I write this discussion, I realized, I might need to create an issue directly in the project [github-label-template](https://github.com/xavierchow/github-label-template) 🤦‍♀️

**Comment by eddiejaoude** [comment, marked as answer]
> I was wondering if we can have one repository which has the JSON file and apply it to all the repositories that a person or an organization owns.

oh yes we could so do this 👍 , we could have a custom GitHub Action that contains this `json` file and adds it to that repo it is included in - this will make it much easier for everyone to use it 💥

**Reply by eddiejaoude** [reply]
I created a repo for this 🤓 and we can start building it up and moving it over https://github.com/EddieHubCommunity/gh-action-open-source-labels
